src/migrations/m230214_071325_add_return_path_2_app_dmstr_contact_template_table.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+use yii\db\Migration;
+
+class m230214_071325_add_return_path_2_app_dmstr_contact_template_table extends Migration
+{
+    public function safeUp()
+    {
+
+        $this->addColumn('app_dmstr_contact_template','return_path', $this->string()->null()->defaultValue(null)->after('to_email'));
```

Comment:
Please use {{%dmstr_contact_template}} instead of specifying the prefix directly. This could lead to potential side-effects

Reply:
Good point, i've just copied the "wrong" existing migration: https://github.com/dmstr/yii2-contact-module/blob/master/src/migrations/m201012_141325_alter_app_dmstr_contact_template_table.php
;-)